emsurfer/src/styles.css

```diff
@@ -0,0 +1,6 @@
+/* You can add global styles to this file, and also import other style files */
+.blue-text{
```

Comment:
[yellow]
Analyze moving this either to a lower level style sheet or even remove and just go with bootstrap stuff

Reply:
Done

Reply:
[yellow]
It seems it's empty now, can we remove it or do we need to leave it for some reason?

Reply:
Done